hal/halx86/acpi/madt.c

```diff
@@ -64,59 +49,35 @@ HalpParseApicTables(
     MadtTable = HalAcpiGetTable(LoaderBlock, APIC_SIGNATURE);
     if (!MadtTable)
     {
-        DPRINT01("MADT table not found\n");
+        DPRINT("MADT table not found\n");
```

Comment:
Not keeping DPRINT1 for these errors?

Reply:
I concur: `DPRINT01` is intended to become `DPRINT1`.